langchain/src/document_loaders/web/notionapi.ts

```diff
@@ -1,11 +1,20 @@
-import { Client, isFullPage, iteratePaginatedAPI } from "@notionhq/client";
 import {
-  GetPageResponse,
+  Client,
+  isFullBlock,
+  isFullPage,
+  iteratePaginatedAPI,
+} from "@notionhq/client";
+import {
+  BlockObjectResponse,
   PageObjectResponse,
 } from "@notionhq/client/build/src/api-endpoints.js";
 import { NotionToMarkdown } from "notion-to-md";
 import { getBlockChildren } from "notion-to-md/build/utils/notion.js";
 
+import {
```

Comment:
If you're just using the types, let's actually use `import type`.

Are these not exported directly from `notion-to-md`?

Reply:
No, unfortunately notion-to-md only exports the NotionToMarkdown class, I had a go at TypeFu to get infer types from the class methods but it was a lot more messy than importing them from the build folder, though I did remove the redundant `index.js`

Reply:
I might send them a PR to export the types and `getBlockChildren` directly from notion-to-md